cogs/help.py

```diff
@@ -3,29 +3,52 @@
 import nextcord
 
 
-class HelpView(ui.View):
-    def __init__(self):
-        super().__init__(timeout=None)
-    # Nextcord help
-    @ui.button(label="Nextcord help", style=ButtonStyle.red, custom_id="help:nextcord")
-    async def nextcord_help(self, _, interaction: Interaction):
-        await self.create_help_thread("Nextcord", interaction)
-    @ui.button(label="D.py libs help", style=ButtonStyle.gray, custom_id="help:dpylibs")
-    async def nextcord_lib_help(self, _, interaction: Interaction):
-        await self.create_help_thread("Discord.py libraries", interaction)
+class HelpView(nextcord.ui.View):
+    
+    @nextcord.ui.button(label='Nextcord Help?', style=nextcord.ButtonStyle.green)
+    async def nextcord_(self, button: nextcord.ui.Button, interaction: nextcord.Interaction):
+        view = Confirmation('Nextcord')
+        await interaction.response.send_message('Click "Yes!" to create the thread.', view=view, ephemeral=True)
+        
+    
+    @nextcord.ui.button(label='Discord.py libs Help?', style=nextcord.ButtonStyle.blurple)
+    async def dpy(self, button: nextcord.ui.Button, interaction: nextcord.Interaction):
+        view = Confirmation('D.py')
+        await interaction.response.send_message('Click "Yes!" to create the thread.', view=view, ephemeral=True)
+        
+    
+    @nextcord.ui.button(label='Python Help?', style=nextcord.ButtonStyle.red)
+    async def python(self, button: nextcord.ui.Button, interaction: nextcord.Interaction):
+        view = Confirmation('Python')
+        await interaction.response.send_message('Click "Yes!" to create the thread.', view=view, ephemeral=True)
+        
+   
+class Confirmation(nextcord.ui.View): 
+    
+    def __init__(self, channel_name):
+        super().__init__()
+        self.name = channel_name       
+        
+    @nextcord.ui.button(label='Yes!', style=nextcord.ButtonStyle.success)
+    async def confirm(self, button: nextcord.ui.Button, interaction: nextcord.Interaction):
+        await self.create_help_thread(name=self.name, interaction=interaction)
```

Comment:
Don't create the thread with the python or those kinds of stuff, I'd recommend doing this.

First, let the user select the button, then ask him what he needs help for, and then send it and add people with pings.

Reply:
"what he needs help for, and then send it and add people with pings." What do you mean by then send it? BTW I just added the confirm buttons and changed what was necessary to add the buttons, the rest is the same as the previous code.